tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -1269,7 +1270,15 @@ private string GetBranchName()
         }
 
         var gitOutput = GitCommandHelper.RunGitCommand(_workingDirectory, "branch --show-current", MetricTags.CIVisibilityCommands.GetBranch);
-        return gitOutput?.Output.Replace("\n", string.Empty) ?? string.Empty;
+        var res = gitOutput?.Output.Replace("\n", string.Empty) ?? string.Empty;
```

Comment:
You should probably handle Windows too? 🤔 
```suggestion
        var res = gitOutput?.Output
            .Replace("\r\n", string.Empty)
            .Replace("\n", string.Empty)
             ?? string.Empty;
```